bundle/compliance/cis_aws/rules/cis_1_7/rule.rego

```diff
@@ -0,0 +1,29 @@
+package compliance.cis_aws.rules.cis_1_7
+
+import data.compliance.lib.common
+import data.compliance.policy.aws_iam.common as iam_common
+import data.compliance.policy.aws_iam.data_adapter
+
+default rule_evaluation = false
+
+# Eliminate use of the 'root' user for administrative and daily tasks
+finding = result {
+	# filter
+	data_adapter.is_root_user
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(rule_evaluation),
+		{"IAM User:": data_adapter.iam_user},
+	)
+}
+
+rule_evaluation {
+	not iam_common.are_credentials_within_duration(data_adapter.active_access_keys, "last_access", "24h")
+	not iam_common.are_credentials_within_duration([data_adapter.iam_user], "last_access", "24h")
+}
+
+rule_evaluation {
+	count(data_adapter.active_access_keys) == 0
+	not iam_common.are_credentials_within_duration([data_adapter.iam_user], "last_access", "24h")
+}
```

Comment:
should be a separate audit function?

Reply:
My general thought about audits is that they will be common among multiple rules.
Currently, I don't think this logic will be shared across multiple rules.

Reply:
Moved to a new audit.